examples/interop-cpp/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "interop-cpp"
 version = "0.1.0"
-authors = ["fi3 <email@email.org>"]
```

Comment:
I think we need @Fi3 approval for those author changes..

Reply:
..why do we actually need to change this?

Reply:
I will wait for @Fi3  confirmation on this.

Reply:
He confirmed on this